Makefile

```diff
@@ -38,6 +38,7 @@ build:
 
 package:
 	cd $(RELEASE_DIR); \
+	tar cvfz $(RELEASE_NAME)-debug.tar.gz dragonfly ../LICENSE.md; \
```

Comment:
This is the actual change. Essentially we are storing a -debug.tar.gz wih the dragonfly binary before its stripped of symbols and then expect the release pipeline to also publish these files.

Reply:
Why there is a whitespace change? Can you please remove unnecessary changes?

Reply:
Sure! Linter did its thing!

Reply:
Fixed!

Reply:
ok, just to make sure the naming is not misleading, I suggest that we call it `$(RELEASE_NAME)-dbgsym.tgz`

Reply:
Done!